torchreid/models/ml_decoder.py

```diff
@@ -70,6 +71,23 @@ def forward(self, tgt: Tensor, memory: Tensor, tgt_mask: Optional[Tensor] = None
         return tgt
 
 
+@torch.jit.script
+class GroupASL(object):
```

Comment:
This class could be derived from nn.Module to preserve pytorch style, but it's up to you.